raft/src/test/java/org/apache/kafka/raft/MockLogTest.java

```diff
@@ -594,6 +594,102 @@ public void testDoesntTruncateFully() throws IOException {
         assertFalse(log.truncateToLatestSnapshot());
     }
 
+    @Test
```

Comment:
If you update `KafkaMetadataLogTest` based on my comments let's update these tests too.

Reply:
done.